defopt.py

```diff
@@ -158,7 +166,9 @@ class _NoTypeFormatter(_Formatter):
     show_types = False
 
 
-def _populate_parser(func, parser, parsers, short):
+def _populate_parser(func, parser, parsers, **kwargs):
+    short = kwargs['short']
+    strict_kwonly = kwargs['strict_kwonly']
```

Comment:
That sounds reasonable in general, but in this case you end up doing more work packing and unpacking than you save. In this particular instance I'd prefer you to change it back.